test/env/update_local_registry.sh

```diff
@@ -19,43 +19,43 @@ contrail-operator/engprod-269421/contrail-operator:master.latest
-contrail-nightly/contrail-controller-webui-job:2008.10
-contrail-nightly/contrail-controller-webui-web:2008.10
-contrail-nightly/contrail-kubernetes-kube-manager:2008.10
-contrail-nightly/contrail-kubernetes-cni-init:2008.10
-contrail-nightly/contrail-node-init:2008.10
-contrail-nightly/contrail-nodemgr:2008.10
-contrail-nightly/contrail-controller-config-dnsmasq:2008.10
-contrail-nightly/contrail-analytics-api:2008.10
-contrail-nightly/contrail-analytics-collector:2008.10
-contrail-nightly/contrail-analytics-query-engine:2008.10
-contrail-nightly/contrail-controller-config-devicemgr:2008.10
+contrail-nightly/contrail-command:master.1417-ubi
+contrail-nightly/contrail-controller-config-api:master.1417-ubi
+contrail-nightly/contrail-controller-config-devicemgr:master.1417-ubi
+contrail-nightly/contrail-controller-config-schema:master.1417-ubi
+contrail-nightly/contrail-controller-config-svcmonitor:master.1417-ubi
+contrail-nightly/contrail-controller-control-control:master.1417-ubi
+contrail-nightly/contrail-controller-control-dns:master.1417-ubi
+contrail-nightly/contrail-controller-control-named:master.1417-ubi
+contrail-nightly/contrail-controller-webui-job:master.1417-ubi
+contrail-nightly/contrail-controller-webui-web:master.1417-ubi
+contrail-nightly/contrail-kubernetes-kube-manager:master.1417-ubi
+contrail-nightly/contrail-kubernetes-cni-init:master.1417-ubi
+contrail-nightly/contrail-node-init:master.1417-ubi
+contrail-nightly/contrail-nodemgr:master.1417-ubi
+contrail-nightly/contrail-controller-config-dnsmasq:master.1417-ubi
+contrail-nightly/contrail-analytics-api:master.1417-ubi
+contrail-nightly/contrail-analytics-collector:master.1417-ubi
+contrail-nightly/contrail-analytics-query-engine:master.1417-ubi
+contrail-nightly/contrail-controller-config-devicemgr:master.1417-ubi
+contrail-nightly/contrail-controller-config-api:2011.55-ubi
+contrail-nightly/contrail-controller-config-devicemgr:2011.55-ubi
+contrail-nightly/contrail-controller-config-schema:2011.55-ubi
+contrail-nightly/contrail-controller-config-svcmonitor:2011.55-ubi
+contrail-nightly/contrail-controller-control-control:2011.55-ubi
+contrail-nightly/contrail-controller-control-dns:2011.55-ubi
+contrail-nightly/contrail-controller-control-named:2011.55-ubi
+contrail-nightly/contrail-controller-webui-job:2011.55-ubi
+contrail-nightly/contrail-controller-webui-web:2011.55-ubi
+contrail-nightly/contrail-kubernetes-kube-manager:2011.55-ubi
+contrail-nightly/contrail-kubernetes-cni-init:2011.55-ubi
+contrail-nightly/contrail-node-init:2011.55-ubi
+contrail-nightly/contrail-nodemgr:2011.55-ubi
+contrail-nightly/contrail-controller-config-dnsmasq:2011.55-ubi
+contrail-nightly/contrail-analytics-api:2011.55-ubi
+contrail-nightly/contrail-analytics-collector:2011.55-ubi
+contrail-nightly/contrail-analytics-query-engine:2011.55-ubi
+contrail-nightly/contrail-controller-config-devicemgr:2011.55-ubi
 contrail-nightly/contrail-command:2011.2
```

Comment:
Here is command second time without ubi is that expected?

Reply:
nope, I missed that

Reply:
fixed